pkgs/applications/audio/scream/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, lib, fetchFromGitHub, cmake, pkg-config
+, alsaSupport ? true, alsaLib
+, pulseSupport ? false, libpulseaudio
```

Comment:
```suggestion
, pulseSupport ? config.pulseaudio or false, libpulseaudio
```

Reply:
Referencing `config` inside the derivation feels a bit odd, and other packages seem to set it in all-packages.nix, so not sure about doing this?

Reply:
Looking at more packages define it in the package itself: https://search.nix.gsc.io/?q=config.pulseaudio&i=nope&files=&repos=

It makes it clear when the option changes without changing files.